docs/Using Fleet/Scripts.md

```diff
@@ -0,0 +1,67 @@
+# Scripts
```

Comment:
The script run as the `root` user right? (Do we want to document such a thing?)

Reply:
It's called out here: https://github.com/fleetdm/fleet/pull/13807/files#diff-72c4e35fedfd42f829d7eeab07a22a64631de796189a108351744bbd42c924f8R7

But good feedback. Maybe we need to be more explicit?

Reply:
No that's fine, my eyes missed what's inside the parenthesis :)